frontend/src/views/teacher/submissions/ShowReviews.vue

```diff
@@ -0,0 +1,101 @@
+<template>
```

Comment:
Is this  component in the student and teacher contexts? Locate under submissions...